KPI/views/mobile_usage_2021.view.lkml

```diff
@@ -2,31 +2,55 @@ view: mobile_usage_2021 {
   derived_table: {
     sql:
       with
-        base as (
-      select
-          submission_date,
-          app_name,
-          canonical_app_name,
-          sum(dau) as dau,
-          sum(wau) as wau,
-          sum(mau) as mau,
-          sum(cdou) as cdou
-      from
-          ${mobile_usage_fields.SQL_TABLE_NAME} AS mobile_usage_fields
-      where
-          {% condition mobile_usage_2021.campaign %} campaign {% endcondition %}
-          AND {% condition mobile_usage_2021.channel %} channel {% endcondition %}
-          AND {% condition mobile_usage_2021.country %} country {% endcondition %}
-          AND {% condition mobile_usage_2021.country_name %} country_name {% endcondition %}
-          AND {% condition mobile_usage_2021.distribution_id %} distribution_id {% endcondition %}
-          AND {% condition mobile_usage_2021.id_bucket %} id_bucket {% endcondition %}
-          AND {% condition mobile_usage_2021.os %} os {% endcondition %}
-      group by 1,2,3 )
+      dau as (
+        select
+            submission_date,
+            app_name,
+            canonical_app_name,
+            sum(dau) as dau,
+            sum(wau) as wau,
+            sum(mau) as mau,
+            sum(cdou) as cdou
+        from
+            ${mobile_usage_fields.SQL_TABLE_NAME} AS mobile_usage_fields
+        where
+            {% condition mobile_usage_2021.campaign %} campaign {% endcondition %}
+            AND {% condition mobile_usage_2021.channel %} channel {% endcondition %}
+            AND {% condition mobile_usage_2021.country %} country {% endcondition %}
+            AND {% condition mobile_usage_2021.country_name %} country_name {% endcondition %}
+            AND {% condition mobile_usage_2021.distribution_id %} distribution_id {% endcondition %}
+            AND {% condition mobile_usage_2021.id_bucket %} id_bucket {% endcondition %}
+            AND {% condition mobile_usage_2021.os %} os {% endcondition %}
```

Comment:
We haven't exposed any of these other filters in the dash, do we want to?

Reply:
So right now the table I'm drawing the new profiles from really only has the `country` column that matches the columns from the mobile (DAU) KPI dataset. so filtering on anything other than country would filter the dau/cdou charts but not the new profiles. Once we have the first_seen_date-based new profile counts, we should add them back to the dash.